bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewRepeaterHandler.java

```diff
@@ -99,11 +100,10 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
             switch (channelUID.getId()) {
                 case CHANNEL_REPEATER_COLOR:
-                    if (command instanceof HSBType) {
+                    if (command instanceof HSBType hsbCommand) {
```

Comment:
So I guess you are not planning to cherry pick back port to OH v3.4.x ;)

Reply:
No, the code is perfectly fine in 3.4.x already. In 4.0 we have a deprecation warning and in the future we would have to refactor this anyway, since the method will probably be removed.